students/olszewski_bartosz/lesson_05_lists/numbers_filter.py

```diff
@@ -0,0 +1,16 @@
+def collecting_list_elements():
+    a = []
+    while True:
+        try:
+            a.append(int(input('Give a list element ')))
+        except ValueError:
```

Comment:
Please try to avoid implementing 'success flow' by catching exceptions.

You have probably mixed two exercises: number filter and age calculator